helm-charts/aerospike-cluster/templates/aerospike-cluster-cr.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   image: {{ .Values.image.repository | default "aerospike/aerospike-server-enterprise" }}:{{ .Values.image.tag | default "5.5.0.7" }}
 
   # Multi pod per host
-  multiPodPerHost: {{ .Values.multiPodPerHost | default true }}
+  multiPodPerHost: {{ .Values.multiPodPerHost }}
```

Comment:
Perhaps we can set it to false by default   multiPodPerHost: {{ .Values.multiPodPerHost | default false }}
As it seems to indicate that it should be set true if we want it (so probably default should have been false):
| `multiPodPerHost` | Set this to `true` to allow scheduling multiple pods per kubernetes node | `true` |

Reply:
Agreed! I put the default to false 🙂